text/0000-create-repo-issues.md

```diff
@@ -0,0 +1,101 @@
+
+Given an RFC PR #002, which discusses a change relevant to the `pack CLI` to add in a `pack my-bags` command. After discussion in Working Group meetings and votes from the Core Team, the RFC is placed in FCP for 7 days. At the end of those, the author should open an issue in the `pack CLI`, which tags the RFC PR. For example, 
+
+```
+### Description
+
+To enable easier traveling, we would like to add a `pack my-bags` command
+
+### Proposed solution
+This issue is a request to implement the proposed solution as detailed in #buildpacks/rfcs#002
+
+### Additional context
+- This feature should be documented as part of standard pack CLI commands documentation.
+```
+
+Upon seeing that issue created, a member of the core team will merge it in. 
+
+# Drawbacks
+[drawbacks]: #drawbacks
+N/A
+
+![Sloth GIF](https://giphy.com/gifs/3NtY188QaxDdC)
+
+# Alternatives
+[alternatives]: #alternatives
+
+- What other designs have been considered?
+    * Core member makes the issue(s) &rarr; This adds extra responsibility when merging issues, and the best person to create the initial issue is the author
+    * Anarchy
+
+# Prior Art
+[prior-art]: #prior-art
+
+- [LSST DM RFC Process](https://developer.lsst.io/v/DM-5063/processes/decision_process.html#adopting-an-rfc) &rarr; `When consensus is established the Assignee should create a set of tickets that specify the implementation work, and then mark the RFC as Adopted in JIRA.`
+- [Rust RFC Process](https://github.com/rust-lang/rfcs#implementing-an-rfc) &rarr; `Every accepted RFC has an associated issue tracking its implementation in the Rust repository; thus that associated issue can be assigned a priority via the triage process that the team uses for all issues in the Rust repository.`
+- [EmberJS RFC Process](https://github.com/emberjs/rfcs#the-rfc-life-cycle) &rarr; `Once an RFC becomes active the relevant teams will plan the feature and create issues in the relevant repositories.`
+
+A number of other RFC processes don't define who creates the issues, though they do emphasize that while the author of an RFC is not obligated to implement it, a given RFC having been accepted implies nothing about what priority is assigned to its implementation, nor whether anybody is currently working on it.
+Some of those include: 
+* [GatsbyJS](https://github.com/gatsbyjs/gatsby/tree/master/rfcs#the-rfc-life-cycle)
+* [NixOS](https://github.com/NixOS/rfcs#the-rfc-life-cycle)
+* [Yarn](https://github.com/yarnpkg/rfcs#the-rfc-life-cycle)
+* [React](https://github.com/reactjs/rfcs/blob/master/README.md#the-rfc-life-cycle)
+
+[Swift](https://github.com/apple/swift-evolution/blob/master/process.md#preparing-an-implementation) requires that an RFC be coupled with a PR for its implementation.
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+# Spec. Changes (OPTIONAL)
```

Comment:
This detail is great, but as a FYI the RFC process is not part of the spec so would not need a spec change.